### PR TITLE
Improve the contributing guide

### LINKED ISSUE #57: Improve the contributing guide
Sections about testing and publishing should be checked and improved.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Improved the contributing guide. Closes #57.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ npm run changelog
 
 #### Packaging the component
 
+**Note** This step is optional in the release process (the script is executed anyway by the postversion hook) but it's required to test the package used as NPM dependency.
```
Comment (oleq) [inline README.md]:
```suggestion
**Note** This step is optional in the release process (the script is executed anyway by the postversion hook) but it is required to test the package used as an NPM dependency.
```

File: README.md [hunk 2]
```diff
@@ -88,13 +90,15 @@ creates a package in the `./dist` directory, which can be then published in the
 
 #### Testing the package before releasing
 
-After generating the package, create a symlink to the `ckeditor5-angular/dist` package directory to test it in another (third–party) Angular project:
+To test the `ckeditor5-angular` package, first bootstrap an empty angular package using [ng new] (https://angular.io/cli/new) and add the `<ckeditor>` component using the [guide](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html) or by cloning and installing the https://github.com/ma2ciek/ckeditor5-angular-test.
```
Comment (oleq) [inline README.md]:
```suggestion
To test the `ckeditor5-angular` package, first bootstrap an empty Angular package using [`ng new`] (https://angular.io/cli/new) and add the `<ckeditor>` component by following the [guide](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html) or by cloning and installing the https://github.com/ma2ciek/ckeditor5-angular-test.
```

Comment (oleq) [reply README.md]: Also, we cannot link to external repositories (non–cksource/ or –ckeditor/) in the official documentation. If we mention that one (https://github.com/ma2ciek/ckeditor5-angular-test), we must fork/move it to the organization.

**Comment by ma2ciek** [comment]
I removed information about cloning *testing package* since testing would be the best just by following our docs for the angular integration.

**Pull request merged**